changelog.md

```diff
@@ -72,6 +72,10 @@
 - [FIXED] Method QueryInterface.bulkDelete no longer working when the model parameter is missing. (PostgreSQL) [#5615](https://github.com/sequelize/sequelize/issues/5615)
 - [ADDED] Context and custom options for deep creation
 - [FIXED] Dates with millisecond precision are inserted correctly in MySQL [#5855](https://github.com/sequelize/sequelize/pull/5855)
+- [ADDED] Where conditions on Instance.save
+- [ADDED] NotAffectedRows error on Postgres on update when option ensureAffectedRows is set to true
```

Comment:
Just single `NotAffectedRows` change log without any dialect name will be sufficient